modules/rubiconBidAdapter.js

```diff
@@ -176,166 +286,57 @@ export const spec = {
   buildRequests: function (bidRequests, bidderRequest) {
     // separate video bids because the requests are structured differently
     let requests = [];
-    const videoRequests = bidRequests.filter(bidRequest => bidType(bidRequest) === 'video').map(bidRequest => {
-      bidRequest.startTime = new Date().getTime();
-
-      const data = {
-        id: bidRequest.transactionId,
-        test: config.getConfig('debug') ? 1 : 0,
-        cur: ['USD'],
```

Comment:
Before we always passed cur USD since rubicon only bids in USD.

I do not see that being passed now.

I am guessing the ORTB Converter will pass `CUR` value if it is defined by prebid, but we may not want this, so we might want to overwrite it and just hardcode `USD` since that was the logic before.

Let's ask Bret perhaps.

Reply:
If you don't see it passed, it's probably because you didn't build pbjs with the currency module. The Ortb library will overwrite the specified currency (line 226) only if the currency module is enabled. Otherwise, no currency at all.

Reply:
Right, but this is changing the behavior, which we should make sure PBS will continue to do the same.

It ALWAYS sends and returns USD now, so our adapter I think assumes USD comes form PBS and Fastlane.

If PBS returns some other currency for some reason we might hurt ourselves.

@bretg

BEFORE:

Rubicon Bid Adapter ALWAYS sent `cur: ['USD']`

AFTER:

Rubicon Bid Adapter will send whatever the currency module is set to

What should we do.

Reply:
I think we should overwrite and always pass `cur: ['USD']` to maintain the same thing as before.

Reply:
This is a question for bret - i'll ask in chat.

Reply:
- if the currency module is not added to pbjs, currency is missing (and pbs will assume USD).
- If the currency module is added, the ortb converter will always add `cur: usd` . There's no way a publisher can change this (if there is, there's a bug in ortb conversion library). That's the sense of line 225 of incoming change: https://github.com/rubicon-project/Prebid.js/pull/62/files#diff-e4dbb62654fc2d8a7b5e6c2c66e0bae395cd945c1985f1509fd4484ecc79fa83R225